.docker/ci/Dockerfile

```diff
@@ -1,50 +1,81 @@
-# moveit/moveit:melodic-ci
```

Comment:
Let's keep this file intact as it is already setup with dockerhub and travis. Perhaps we could move your changes to `.docker/circle-ci/Dockerfile`. Once that is done I am comfortable giving this a +1. 

On a related note, if you have access to the ros-planning repo, I would recommend pushing your branch to ros-planning/moveit2 so that we can test that everything works with Dockerhub. I did the same thing for my PR #46 and it has been really helpful for iterating quickly.

Reply:
I've restored the old Dockerfile in 6d642ca , but reminded I had to have new Dockerfile at the root of the project so that the build context may encompass the entire repo that it copies from. I've added a .dockerignore file to mitigate build cache breaks from git fetches or editing of the Dockerfile itself.